IceCubesApp/Resources/Localization/Localizable.xcstrings

```diff
@@ -65749,6 +65749,23 @@
         }
       }
     },
+    "status.row.is-thread" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Continues a thread"
+          }
+        },
+        "fr" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Complète un fil"
+          }
+        }
+      }
+    },
```

Comment:
Unfortunately you need to add all translation with the default English. If not it'll show the label key :/